code/modules/client/preferences.dm

```diff
@@ -686,23 +686,26 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 			dat += "<tr><td colspan=4><hr></td></tr>"
 			dat += "<tr><td><b>Name</b></td>"
-			dat += "<td><b>Cost</b></td>"
+			if(LC.category != "Donator" && CONFIG_GET(flag/donator_items))
```

Comment:
Won't this work unexpectedly when donator items are disabled by config? It looks like it will remove cost text when the config isn't turned on.